examples/with-snowpack/babel.config.json

```diff
@@ -3,7 +3,7 @@
     [
       "@babel/preset-env",
       {
-        "targets": "> 2%, not dead, not ie > 0",
+        "targets": "> 2%, supports es6-module",
```

Comment:
Oh interesting, but this is assuming no post-bundler, right? Just if you used snowpack by itself?

Reply:
I thought it could be interesting to remove `not ie > 0` and `not dead` as they are both included in `> 2%` now, and to add `supports es6-module` to be explicit in this SnowPack demo, because we are using native ESM. But it’s not really needed as `> 2%` already only includes ESM-enable browsers.